app.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const Hapi = require('hapi');
+const dotenv = require('dotenv').config();
+
+const port = process.env.PORT || 4040;
+
+const server = Hapi.server({
+    port,
```

Comment:
This will do that if the port is set on the prod server. Otherwise it falls back to `4040`. We have a `.env` for all of the variables we use on the process like API keys etc.

Reply:
cool cool.